Cargo.toml

```diff
@@ -12,10 +12,11 @@ categories = ["os", "os::linux-apis", "network-programming", "hardware-support"]
 exclude = [".github/"]
 
 [features]
-default = ["backend-kvm", "backend-sev", "backend-sgx"]
+default = ["backend-kvm", "backend-sev", "backend-sgx", "backend-direct"]
```

Comment:
Do all of these really need to be on by default? The whole reason that we're splitting out the exec into two crates is to avoid unnecessarily building wasmtime twice, so why bother if we're going to build it twice anyway?

Reply:
in our CI jobs we turn off the default features. What backend is on by default should be discussed separately in a broader group.